testnets/airchainstestnet/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "airchainstestnet",
+  "assets": [
+    {
+      "description": "The native staking token of Airchains Testnet.",
+      "denom_units": [
+        {
+          "denom": "amf",
+          "exponent": 0
+        },
+        {
+          "denom": "amf",
+          "exponent": 6
+        }
+      ],
+      "base": "amf",
+      "name": "Airchains",
+      "display": "amf",
+      "symbol": "AMF",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/airchainstestnet/images/airchains-mono.png",
```

Comment:
this is not the correct file name (-mono). The PNG is -logo. I'd suggest renaming the visually-identical image files to be the same. Then, but sure the include all images in the `images` array. The logo_URIs is an optional legacy property.